bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/ActionImpl.java

```diff
@@ -31,23 +33,28 @@
  * @author Ana Dimova - Initial Contribution
  * @author Vasil Ilchev - Initial Contribution
  */
-public class ActionImpl extends ModuleImpl<ActionHandler>implements Action, ConnectedModule, SourceModule {
+public class ActionImpl extends Action implements ConnectedModule, SourceModule {
```

Comment:
I think the name ActionImpl is not good, because Action now already is an implementation. What is the difference of Action and ActionImpl? I guess it´s the handler. Maybe something like "RuntimeAction"?